enterprise/admin/forms.py

```diff
@@ -185,7 +263,8 @@ def get_catalog_options(self):
         Once retrieved, these name pairs can be used directly as a value
         for the `choices` argument to a ChoiceField.
         """
-        catalogs = get_all_catalogs(self.user)
+        catalog_api = CourseCatalogApiClient(self.user)
```

Comment:
@mattdrayer Question to @saleem-latif as author of this code - I only switched it to use `CourseCatalogApiClient` as part of refactoring.